projects/WTI-UI/.classpath

```diff
@@ -1,9 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.web.container/WTI-API"/>
-	<classpathentry kind="lib" path="/pc2v9/vendor/lib/junit-4.11.jar"/>
-	<classpathentry kind="src" path="/WTI-API"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jst.j2ee.internal.web.container/WTI-API"/>
+	<classpathentry kind="lib" path="/pc2v9/vendor/lib/junit-4.11.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>

```

Comment:
In the future this change should be done in its own PR.  
It is not obvious from the PR that a change in the classpath needs to be done.

Reply:
In principle I agree, and I'll consider doing such, "in the future".  That "future" would be at a time when it no longer is the case that changes submitted for review sit up to seven  weeks with zero comments (as has already happened).   For now, it's hard enough to move forward without further subdividing the work into additional PRs that require separate reviews.

Reply:
Doug and I had a lengthy phone conversation about this issue.

* His argument (he can correct me if I've misstated it) is that having the packageWTI.xml code modify the pc2v9.ini file could lead to unintended consequences down the road if the packageWTI.xml code was changed.
* While I don't disagree that this COULD happen, my argument is that
  * such a thing could happen ANYWAY, and
  * the approach suggested by him suffers from the known-bad-practice of maintaining two copies of a single source code file.

Neither of us was able to convince the other that their argument had substantially more weight, leaving an impasse.  However, in the process of the discussion it was pointed out that the EWTeam project is ALREADY doing things the way Doug suggests:  it maintains a separate project-specific copy of pc2v9.ini within the project.

Given that, and in order to resolve the conflict, I agree to accept Doug's review recommendation.  I will submit a PR update which removes the pc2v9.ini editing from packageWTI.xml, and also updates the WTI-specific copy of pc2v9.ini with the appropriate initial values (the ones which the editing was previously adding to the pc2v9.ini file).

I believe this should resolve this issue.